include/zephyr/bluetooth/mesh/access.h

```diff
@@ -145,19 +145,23 @@ extern "C" {
  *  @param _mods      Array of models.
  *  @param _vnd_mods  Array of vendor models.
  */
-#define BT_MESH_ELEM(_loc, _mods, _vnd_mods)        \
-{                                                   \
-	.loc              = (_loc),                 \
-	.model_count      = ARRAY_SIZE(_mods),      \
-	.vnd_model_count  = ARRAY_SIZE(_vnd_mods),  \
-	.models           = (_mods),                \
-	.vnd_models       = (_vnd_mods),            \
+#define BT_MESH_ELEM(_loc, _mods, _vnd_mods)		\
+{							\
+	.rt		  = (void *)(void *[]) { NULL },\
+	.loc              = (_loc),			\
+	.model_count      = ARRAY_SIZE(_mods),		\
+	.vnd_model_count  = ARRAY_SIZE(_vnd_mods),	\
+	.models           = (_mods),			\
+	.vnd_models       = (_vnd_mods),		\
 }
 
 /** Abstraction that describes a Mesh Element */
 struct bt_mesh_elem {
-	/** Unicast Address. Set at runtime during provisioning. */
-	uint16_t addr;
+	/** Mesh Element runtime information */
+	struct {
```

Comment:
```suggestion
	struct runtime_ctx {
```

Reply:
Add `bt_mesh_` prefix more suitable.

Reply:
The same code has been approved by you in the previous PR: https://github.com/zephyrproject-rtos/zephyr/pull/65152
Should we change this for bt_mesh_model then?

Reply:
Done.